Entities/Vehicles/Boats/Raft.as

```diff
@@ -1,4 +1,4 @@
-// Boat logic
+#include "GenericButtonCommon.as"
```

Comment:
This does not appear to be used?

Reply:
It is not used. This line can be removed safely.